lib/fhmongodb.js

```diff
@@ -303,10 +303,19 @@ Database.prototype.remove = function (collectionName, id, callback) {
   var self = this;
 
   this.db.collection(collectionName, function (err, collection) {
-    if (null !== err) return callback(err, null);
+    if (null !== err){
+      return callback(err, null);
+    }
+
+    // if a GUID id is passed as strings, convert it to an ObjectID 
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the useless trailing whitespaces at the end of this line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://metrics.skunkhenry.com/coding_rules#rule_key=javascript%3ATrailingWhitespace)